### PR TITLE
[QTM-536] removes redundant nav tags

### PR DESCRIPTION
**Pull request by alizeleal**
Description: https://jirasoftware.catho.com.br/browse/QTM-536

Review guide:
- [ ] Code review
- [ ] Layout

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -85,8 +89,14 @@ export default class HomePage extends App {
                 </nav>
               </Col>
             </Row>
-            <nav id="hiddenDesktopMenu" className="HiddenMenu HideMenu">
+            <nav id="hiddenDesktopMenu" className="HiddenMenu HideMenu ">
```
Comment (rapahaeru) [inline pages/_app.js:92]: Why the style of fonts were changed?
image

File: pages/_app.js [hunk 2]
```diff
@@ -21,12 +21,16 @@ export default class HomePage extends App {
         hiddenMenu.classList.add('HideMenu');
       }
 
-      if (mobileMenu.classList.contains('HideMenu') && showMobileMenu) {
-        mobileMenu.classList.add('ShowMenu');
-        mobileMenu.classList.remove('HideMenu');
-      } else {
-        mobileMenu.classList.add('HideMenu');
-        mobileMenu.classList.remove('ShowMenu');
+      if (hiddenMenu.classList.contains('MobileMenu') && showMobileMenu) {
```
Comment (rapahaeru) [inline pages/_app.js:24]: The problem with responsiveness is here, but we'll remove the menu function with JS soon.. So keep in this way

**Review by rapahaeru — CHANGES_REQUESTED** [review]
Reviewing this PR is difficult because we should have accomplished the task of reducing the menu function to just CSS sooner

File: pages/_app.js (hunk 1 above)
Comment (alizeleal) [reply pages/_app.js:92]: fixed

**Review by rapahaeru — APPROVED** [review]

**Pull request merged**